data/phones/arm_e_phonetic.phones

```diff
@@ -1,66 +1,94 @@
 # Based on https://en.wikipedia.org/wiki/Armenian_language#Phonology
+# And based on the pronunciation script from Wiktionary: https://en.wiktionary.org/wiki/Module:hy-pronunciation
+#
 # Vowels.
 ɑ
 ɛ
 ə
 i
 ɔ
-o  # <ո> is really [ɔ].
-a  # <ա> is really [ɑ].
 u
+#
+# Older entries might contain [o] or [e]. These have been fixed. You shouldn't find these
+# phones anymore.
+#
+# Long vowels: A sequence of identical heterosyllabic vowels is phonetically pronounced as 
+# one long vowel. Armenian does not have phonemic vowel length. But these are very rare. 
+# The only attested case on Wiktionary is the following.
+ɛː
+# The other types of long vowels are not found on Wiktionary as of Jan 2021. This is 
+# largely because long vowels are mostly in loanwords and borrowings, which Wiktionary 
+# underreports. At some point in the future, the Armenian users might add some entries 
+# which contain these sequences.
+# ɑː əː iː ɔː uː
```

Comment:
If these are accidental gaps, could you please uncomment these and put them on their own lines? The same applies to the "missing" affricates, and the gaps in the file as well.

It's not a problem to have phones in the `.phones` file that don't currently appear on Wiktionary, whereas it will be a problem later on if someone adds a word with, say, /ɑː/, and we accidentally (and unknowingly) filter it out.

Reply:
I fixed the comments